internal/pkg/composable/providers/kubernetes/pod.go

```diff
@@ -492,16 +503,22 @@ func generateContainerData(
 	}
 }
 
-func getHintsMapping(k8sMapping map[string]interface{}, logger *logp.Logger, prefix string, cID string) mapstr.M {
+func getHintsMapping(k8sMapping map[string]interface{}, logger *logp.Logger, prefix string, cID string) (mapstr.M, []mapstr.M) {
 	hintsMapping := mapstr.M{}
+	processorMapping := []mapstr.M{}
+
 	if ann, ok := k8sMapping["annotations"]; ok {
 		annotations, _ := ann.(mapstr.M)
 		hints := utils.GenerateHints(annotations, "", prefix)
 		if len(hints) > 0 {
 			logger.Debugf("Extracted hints are :%v", hints)
 			hintsMapping = GenerateHintsMapping(hints, k8sMapping, logger, cID)
 			logger.Debugf("Generated hints mappings are :%v", hintsMapping)
+
+			processorMapping = utils.GetConfigs(annotations, prefix, "hints/processors")
+			logger.Debugf("Generated Processor mappings are :%v", processorMapping)
```

Comment:
Technically processors are not mapping for the dynamic provider. I would avoid using this term.
```suggestion
			logger.Debugf("Generated Processors are :%v", processorMapping)
```